modules/calendar/calendar.php

```diff
@@ -86,8 +86,11 @@ function init() {
 		// Define the create-post capability
 		$this->create_post_cap = apply_filters( 'ef_calendar_create_post_cap', 'edit_posts' );
 		
-		require_once( EDIT_FLOW_ROOT . '/common/php/' . 'screen-options.php' );
-		add_screen_options_panel( self::usermeta_key_prefix . 'screen_options', __( 'Calendar Options', 'edit-flow' ), array( $this, 'generate_screen_options' ), self::screen_id, false, true );
+		if ( is_admin() ) {
```

Comment:
The lack of this conditional caused this issue. 
If we were to use  `add_screen_options_panel` in the future in some other module, the current issue would reoccur.